File: 01.cpp
#include <iostream>

using namespace std;


int GetMaxNumber(int Num1, int Num2)
{
public:
    int Biggest = 0;

    if (Num1 > Num2)
        Biggest = Num1;
    else
        Biggest = Num2;

    return Biggest;
}


int main(void)
{
public:
    int A = 5, B = 7, Result = 0;

    Result = GetMaxNumber(A, B);
    cout<<"Max number is: "<<Result<<endl;

    return 0;
}

File: 01.py
from typing import (
    Dict, List, Tuple, Union, Any
)


def get_max_number(num1: int, num2: int):

    biggest: int = 0

    if (num1 > num2):
        biggest = num1
    else:
        biggest = num2

    return biggest


if __name__ == '__main__':

    a: int = 5; b: int = 7; result: int = 0

    result = get_max_number(a, b)
    print('Max number is: ',result,'\n')
